website/content/docs/getting-started/connect-to-target.mdx

```diff
@@ -38,51 +38,57 @@ within the authorized session. When you are finished making connections, simply
 It can be annoying to keep accepting host SSH key prompts as the port changes,
 much less having to keep copying and pasting the current port. To make this
 easier, Boundary includes connect helpers that take care of this work for you.
```

Comment:
I was reading this bit and couldn't help but think about the workshops yesterday. It just seems like extra fluff and I wonder if we should keep it.

Perhaps we could have something lighter like "Boundary includes connect helpers that automatically accept host SSH key prompts for you" ?

Reply:
There's also a bit of repetition around as we talk about these helpers in the `ssh` context above - Perhaps we should introduce the connect helpers concept first, and then go into the SSH example?

Reply:
To be honest, I like your rewrite here. It's much more direct and easier to understand